CHANGELOG.md

```diff
@@ -3,6 +3,18 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## v1.1.4dev - [unreleased]
+
+### `Added`
+
+### `Fixed`
+
+- [###]() Added new parameter `annotation_prokka_retaincontigheaders` to allow prokka to retain the original contig headers/locus tag. (by @darcy220606)
```

Comment:
To be updated

Reply:
exactly :D hence why i added it as optional and not default